src/tutorials/annotation.md

```diff
@@ -108,14 +135,20 @@ and links to helpful guides and tutorials with the details.
 
 ### The event annotation process
 
-The goal of event annotation is to provide information about events needed for effective and correct data analysis.
+The goal of event annotation is to provide information about
+events needed for effective and correct data analysis.
 
-Ideally, most of this information should be in a single `events.json` sidecar file located in the root directory of your dataset, where it is easy to find and update.
+Ideally, most of this information should be in a single `events.json` sidecar file
+located in the root directory of your dataset,
+where it is easy to find and update.
 
-An overview of how event annotation works in BIDS, as well as tutorials about using available online tools to facilitate annotation, can be found in the
+An overview of how event annotation works in BIDS as well as tutorials
+about using available online tools to facilitate annotation can be found in the
 [BIDS annotation quickstart](https://hed-examples.readthedocs.io/en/latest/BidsAnnotationQuickstart.html).
```

Comment:
A better link is: 
https://www.hed-resources.org/en/latest/BidsAnnotationQuickstart.html

Reply:
Updated in latest [commit](https://github.com/bids-standard/bids-starter-kit/pull/335/commits/ec93015455d74f60ae0ce5cc70341d16b1c7e609).